src/Tizen.NUI/src/public/BaseComponents/TextEditorEvent.cs

```diff
@@ -160,6 +166,32 @@ public partial class TextEditor
             }
         }
 
+        /// <summary>
+        /// The SelectionChanged event.
```

Comment:
(Minor comment)
Please write the description of each function in detail next time.